src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.controller;
+
+import racingcar.model.Car;
+import racingcar.model.Cars;
+import racingcar.model.RaceGame;
+import racingcar.validation.CarNamesValidation;
+import racingcar.validation.RaceCountsValidation;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingController {
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+
+    public void run() {
+        List<String> carNames = CarNamesValidation.validateCarNames(inputView.readCarNames());
+        int raceCount = RaceCountsValidation.validateRaceCounts(inputView.readRaceCounts());
```

Comment:
1주차 공통 피드백의 "축약하지 않는다" 부분을 참고하시면 좋을 것 같습니다!
CarNamesValidation 클래스에서 수행하는 검증은 CarNames에 대한 중복을 수행할 것이라고 예측할 수 있기에
문맥을 중복하지 않고 메서드 이름을 validate()로 하는건 어떨까요?

Reply:
inputView.readCarNames()와 inputView.readRaceCounts()를 아래 코드처럼 표현하는건 어떻게 생각하시나요? 저는 개인적으로 파라미터로 전달되는 값이 더 명확히 보이고, 가독성이 더 좋다고 생각합니다!
```suggestion
        String raceCounts = inputView.readRaceCounts();
        int raceCount = RaceCountsValidation.validateRaceCounts(raceCounts);
```

Reply:
공통 피드백에 더 신경 써서 3주차에는 나은 코드로 뵙겠습니다! 두 분께서 설명 주신 부분에 대해 제 코드를 다시 확인해 보는 중이에요 말씀하신 것처럼 String `raceCounts`에서 이를 `raceCount`의 인자로 넘겨 주는 것이 코드 복잡도가 떨어진다 생각해요!